tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -71,25 +70,54 @@ void output_certificate(const uint8_t* data, size_t data_len)
             XN_FLAG_COMPAT,
             XN_FLAG_SEP_CPLUS_SPC | XN_FLAG_DUMP_UNKNOWN_FIELDS);
     BIO_free_all(input);
-    printf("\n");
 #endif
     OE_UNUSED(data);
     OE_UNUSED(data_len);
 }
 
-void output_certificate_chain(const uint8_t* data, size_t data_len)
+void decode_certificate_pem(
```

Comment:
Is it more flexible to input file pointer instead of logging?

void decode_certificate_pem(
    FILE* fp,
    const uint8_t* data,
    size_t data_len,)

Reply:
Sure I update to function to take a file pointer.